src/main/java/com/moneylendingapp/services/DefaultUserService.java

```diff
@@ -1,8 +1,11 @@
 package com.moneylendingapp.services;
 
+import com.moneylendingapp.dto.requests.LoginRequest;
 import com.moneylendingapp.dto.requests.SignUpRequest;
+import com.moneylendingapp.dto.responses.LoginResponse;
 import com.moneylendingapp.dto.responses.UserModel;
 
 public interface DefaultUserService {
```

Comment:
Perhaps you should name this UserService and the implementation would be called DefaultUserService